lib/api/2.0/hooks.js

```diff
@@ -0,0 +1,55 @@
+// Copyright 2016, EMC, Inc.
+
+'use strict';
+
+var injector = require('../../../index.js').injector;
+var controller = injector.get('Http.Services.Swagger').controller;
+var hooks = injector.get('Http.Services.Api.Hooks');
+
+/**
+ * @api {get} /api/2.0/hooks GET /
+ * @apiDescription get list of hooks
+ * @apiName hooks-get
+ * @apiGroup hooks
+ */
+var hooksGetAll = controller(function(req) {
+    return hooks.getHooks(req.query);
+});
+
```

Comment:
You forgot another API : ` GET /hooks/:identifier`, GET the specific hook by identifier.